cpp/Platform.Random/RandomHelpers.h

```diff
@@ -2,6 +2,6 @@
 {
     class RandomHelpers
     {
-        public: static readonly System::Random Default = System::Random(System::DateTime::UtcNow::Ticks::GetHashCode());
+        public: static const std::mt19937_64 Default;
```

Comment:
`std::mt19937_64` forward as ref. Remove const

Reply:
Use this declaration 
```cpp
static std::mt19937_64 Default(std::random_device{}());
```

Reply:
> Use this declaration
> static std::mt19937_64 Default(std::random_device{}());

The Default random helper would not be static then. The compiler doesn't like it

Reply:
static similarly global. Why is a problem?
What the compiler says